skada/tests/test_mapping.py

```diff
@@ -171,23 +192,33 @@ def _base_test_new_X_adapt(estimator, da_dataset):
         (LinearOTMappingAdapter()),
         (CORALAdapter()),
```

Comment:
We do not need CORALAdapter twice I guess?

Reply:
I don't know what was the original idea for the test but all of them are used twice in here. I can clean it up

Reply:
removed duplicates